__tests__/seed.test.js

```diff
@@ -44,5 +44,41 @@ describe("GET /api/topics", () => {
           expect(typeof topic.description).toBe("string");
         });
       });
-  })
+  });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("GET200: Endpoint responds with selected article by its ID", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(typeof article.author).toBe("string");
```

Comment:
have a look at .toMatchObject and .objectContaining here for a better way to assert against objects, this is a bit hard to read